### DISCUSSION TITLE
overlayScrollbars with livewire

### DISCUSSION THREAD

**Discussion by imhitt** [livewire/livewire#4400] [1. Help]
Hi,

I am using livewire for realtime chat with overlayScrollbars to change default scrollbar.

Is there a way to use overlayScrollbars with livewire for dynamic element ?

overlayScrollbars - generate deep nested DOM.
wire:ignore - not working because livewire used for render realtime content
wire:ignore.self - not working because it holds data only for parent element but childs will be destroyed after rerender.

**Comment by joshhanley** [comment]
@imhitt yeah I think Livewire and OverlayScrollbars doesn't work very well due to how much scrollbars changes which then Livewire will wipe out.

I have seen something about someone else trying to use it, but I can't remember whether it was in this repos issues and discussion, or on discord, so might be worth doing a search to see if you can find it and see if they solved it or not.

Hope this helps!
